python/ray/data/datasource/datasource.py

```diff
@@ -17,16 +17,16 @@ class Datasource:
     @Deprecated
     def create_reader(self, **read_args) -> "Reader":
         """
-        Deprecated: Implement :meth:`Datasource.get_read_tasks` and
-        :meth:`Datasource.estimate_inmemory_data_size` instead.
+        Deprecated: Implement :meth:`~ray.data.Datasource.get_read_tasks` and
+        :meth:`~ray.data.Datasource.estimate_inmemory_data_size` instead.
```

Comment:
For my own understanding, do I always need to qualify the class name, even if the class is defined in the same module?

Reply:
It depends on how you have Sphinx configured, and the syntax you use. I feel like the simplest thing to do is fully qualify it though.